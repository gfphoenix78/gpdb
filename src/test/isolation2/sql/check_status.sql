select dbid, content, status, role, preferred_role, mode from gp_segment_configuration where status = 'd' or preferred_role != role;
select gp_inject_fault('all', 'reset', dbid) from gp_segment_configuration where status='u';
