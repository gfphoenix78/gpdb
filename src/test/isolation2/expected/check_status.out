select dbid, content, status, role, preferred_role, mode from gp_segment_configuration where status = 'd' or preferred_role != role;
 dbid | content | status | role | preferred_role | mode 
------+---------+--------+------+----------------+------
(0 rows)
select gp_inject_fault('all', 'reset', dbid) from gp_segment_configuration where status='u';
 gp_inject_fault 
-----------------
 Success:        
 Success:        
 Success:        
 Success:        
 Success:        
 Success:        
 Success:        
 Success:        
(8 rows)
